app/views/benefits/RemoveCompanyBenefitStopDateView.scala.html

```diff
@@ -43,7 +43,9 @@
     pageTitle = Some(Messages("tai.currentYearSummary.heading")),
     title = Messages("tai.benefits.ended.stopDate.heading", benefitType, empName),
     authedUser = Some(user),
-    backLinkContent = Some(Messages("tai.back-link.upper"))
+    backLinkId = "backLinkUrl",
```

Comment:
ID's don't need to be changed

Reply:
I've updated it to `backLinkId = "backLink"` but left it explicit as the default value is `backLinkId: String = "returnToChooseTaxYearLink",`